### ISSUE TITLE
How do I use this?

### ISSUE THREAD

**Issue by justinireland** [balloob/react-sidebar#16]
Could you please ELI5 how I would go about including this component with my project? Sorry, I am still new to react and not quite sure how to include 3rd party components.

Currently I am using node/express and rendering everything client side.

**Comment by balloob** [comment]
Please see the example which is included in the repository

**Comment by justinireland** [comment]
I have looked at the examples but I want to be able to require the file as you have it in the readme instructions.

**Comment by jwamsterdam** [comment]
I see this issue is closed, which I think is incorrent. I see very limited overlap between the readme and the examples. For instance, in the examples, I cannot find the same styles as in the readme. Can more instructions be given or examples / readme be updated?
